CoreVisualSchema/test/unit/src/au/gov/asd/tac/constellation/graph/schema/visual/attribute/objects/PlaneNGTest.java

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright 2010-2022 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.graph.schema.visual.attribute.objects;
+
+import java.awt.image.BufferedImage;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author serpens24
+ */
+public class PlaneNGTest {
+        
+    // Create object under test
+    final BufferedImage bufferedImage = new BufferedImage(1, 2, 3);
+    Plane instance;
+
+    public PlaneNGTest() {
+    }
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        System.out.println("PlaneNGTest.");
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+        System.out.println("PlaneNGTest.");
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+        instance = new Plane("label", 1.0f, 2.0f, 3.0f, 4.0f, 5.0f, bufferedImage, 7, 8);
+    }
+
+    @AfterMethod
+    public void tearDownMethod() throws Exception {
+    }
+
+    /**
+     * Test of getImage method, of class Plane.
+     */
+    @Test
+    public void testGetImage() {
+        System.out.println("PlaneNGTest.testGetImage");
+        final BufferedImage value = instance.getImage();
+        assertEquals(value.getWidth(), bufferedImage.getWidth());
+        assertEquals(value.getHeight(), bufferedImage.getHeight());
+        assertEquals(value.getType(), bufferedImage.getType());
+        assertEquals(value.getType(), bufferedImage.getType());
```

Comment:
duplicate?

Reply:
it does look lile 69 and 70 are duplicates .... I'll look and fix.